src/actions/guides/database/querying.cr

```diff
@@ -7,294 +7,526 @@ class Guides::Database::Querying < GuideAction
-    UserQuery.new.age(30.0)
+    ### A gt/lt B
+
+    Find rows where `A` is greater than or equal to `B`.
+
+    `WHERE users.age >= 21`
+
+    ```crystal
+    UserQuery.new.age.gte(21)
     ```
 
-    > Note that this will change in the future but this is the current behavior. In
-    the future only the defined type (`Int32` in this example) will work. You will
-    then need to manually parse it. Don't worry though, Lucky will add some helpers
-    to actions and the model to make this easy and maintainable.
+    Find rows where `A` is greater than `B`.
 
-    ## Ordering results
+    `WHERE users.created_at > '#{1.day.ago}'`
 
-    ````crystal
-    UserQuery.new.age.desc_order
-    # or
-    UserQuery.new.age.asc_order
-    ````
+    ```crystal
+    UserQuery.new.created_at.gt(1.day.ago)
+    ```
 
-    ## Chaining methods
+    Find rows where `A` is less than or equal to `B`.
 
-    You can chain where clauses and most other query methods.
+    `WHERE users.age <= 12`
 
     ```crystal
-    UserQuery.new.age.gt(28).age.desc_order
+    UserQuery.new.age.lte(12)
     ```
 
-    ## Running the query
+    Find rows where `A` is less than `B`.
 
-    The query does not actually hit the database until a method is called to fetch a
-    result or iterate over results.
+    `WHERE users.updated_at < '#{3.months.ago}'`
 
-    Some of the most common methods are:
+    ```crystal
+    UserQuery.new.updated_at.lt(3.months.ago)
+    ```
 
-    * `first`
-    * `find`
-    * `each`
+    ### A in / not in (B)
 
-    For example:
+    Find rows where `A` is in the list `B`.
+
+    `WHERE users.name IN ('Bill', 'John')`
 
     ```crystal
-    # The query is not yet run
-    query = UserQuery.new
-    query.name("Sally")
-    query.age(30)
+    UserQuery.new.name.in(["Bill", "John"])
+    ```
 
-    # The query will run once `each` is called
-    # Results are not cached so a request will be made every time you call `each`
-    query.each do |user|
-      pp user.name
-    end
+    Find rows where `A` is not in the list `B`.
+
+    `WHERE users.name NOT IN ('Sally', 'Jenny')`
+
+    ```crystal
+    UserQuery.new.name.not.in(["Sally", "Jenny"])
     ```
 
-    ## Finding by ID
+    ### A like / iLike B
+
+    Find rows where `A` is like (begins with) `B`.
+
+    `WHERE users.name LIKE 'John%'`
 
     ```crystal
-    id = 1
-    UserQuery.new.find(id)
+    UserQuery.new.name.like("John%")
+    ```
+
+    `WHERE users.name ILIKE 'jim'`
 
-    # You can also chain methods before calling find, first, etc.
-    UserQuery.new.age(30).find(1)
+    ```crystal
+    UserQuery.new.name.ilike("jim")
     ```
 
-    ## Ensuring no results are returned
+    ## Ordering
+
+    Return rows ordered by the `age` column in descending (or ascending) order.
+
+    `SELECT COLUMNS FROM users ORDER BY users.age DESC`
 
     ```crystal
-    UserQuery.new.none
+    UserQuery.new.age.desc_order
+    # or for asc order
+    UserQuery.new.age.asc_order
     ```
```

Comment:
Maybe a heading like `## Negating conditions with NOT`

That shows a few examples of using `not`. `name.not("John")` or `age.not(30)`, `name.not.ilike("Jess%")` just so it is clear it can be used in most where's and also so it shows up in search

Reply:
I have this example `UserQuery.new.name.not.eq("Billy")`. Should this be `name.not("Billy")`?

Reply:
You've got it right with `not.eq` that was my bad :)